// This file has been generated by Py++.

#include "boost/python.hpp"
#include "wrap_osgDB.h"
#include "wrap_referenced.h"
#include "OutputException.pypp.hpp"

namespace bp = boost::python;

void register_OutputException_class(){

    bp::class_< osgDB::OutputException, bp::bases< ::osg::Referenced >, osg::ref_ptr< ::osgDB::OutputException >, boost::noncopyable >( "OutputException", bp::init< std::vector< std::string > const &, std::string const & >(( bp::arg("fields"), bp::arg("err") )) )    
        .def( 
            "getError"
            , (::std::string const & ( ::osgDB::OutputException::* )(  ) const)( &::osgDB::OutputException::getError )
            , bp::return_value_policy< bp::copy_const_reference >() )    
        .def( 
            "getField"
            , (::std::string const & ( ::osgDB::OutputException::* )(  ) const)( &::osgDB::OutputException::getField )
            , bp::return_value_policy< bp::copy_const_reference >() );

}
